### PR TITLE
Fix session deprecation

### PR DESCRIPTION
**Pull request by dannyvw**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | yes 
| Related tickets | partially #335
| License         | MIT

### CODE REVIEW

**Review by Roshyo — APPROVED** [review]

File: src/Bundle/DependencyInjection/Compiler/SessionPass.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace Sylius\Bundle\ResourceBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * TODO Remove after bumping to > Symfony 5.x
```
Comment (lchrusciel) [inline src/Bundle/DependencyInjection/Compiler/SessionPass.php:22]: What about bumping minimal requirements to 4.4 already?

**Comment by lchrusciel** [comment]
Thank you, Danny! :tada:

File: src/Bundle/Controller/FlashHelper.php [hunk 2]
```diff
@@ -89,8 +103,14 @@ private function addFlash(string $type, string $message, array $parameters = [])
             $message = $this->prepareMessage($message, $parameters);
         }
 
+        if ($this->requestStack instanceof SessionInterface) {
```
Comment (lchrusciel) [inline src/Bundle/Controller/FlashHelper.php:106]: Having afterthought, perhaps we should wrap it into our custom class, that would do this logic?

File: src/Bundle/DependencyInjection/Compiler/SessionPass.php (hunk 1 above)
Comment (Roshyo) [reply src/Bundle/DependencyInjection/Compiler/SessionPass.php:22]: If you meant 5.4, then I agree

**Pull request merged**